libraries/src/Installer/Installer.php

```diff
@@ -512,6 +512,17 @@ public function install($path = null)
 		// Run the install
 		$result = $adapter->install();
 
+		/**
+		 * Flush the opcache regardless of result to ensure consistency
+		 *
+		 * In some (most?) systems PHP's CLI has a separate opcode cache to the one used by the web server or FPM process,
+		 * which means running opcache_reset() in the CLI won't reset the webserver/fpm opcode cache, and vice-versa.
+		 */
+		if (function_exists('opcache_reset'))
+		{
+			opcache_reset();
```

Comment:
```suggestion
			\opcache_reset();
```
I think it can be done like suggested because it's a namespaced file. See also https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/components/com_joomlaupdate/restore_finalisation.php#L78-L82

Reply:
@SharkyKZ Or am I wrong?

Reply:
If Nik’s code did it that way then we should - I trust him :/)

Also clearstatcache might also be a good call like his code

Reply:
added clearstatcache and root namespace